<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonNew">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Crea Nuova Lega</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifica Lega Esistente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFixtures">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Genera Calendario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Genera Coppa di Lega</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Apri Lega Esistente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsertLineup">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserisci Formazione</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsertVotes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserisci Voti</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
